config/cockroachdb/sample_wikipedia_config.xml

```diff
@@ -8,11 +8,14 @@
     <username>root</username>
     <password></password>
     <isolation>TRANSACTION_SERIALIZABLE</isolation>
-    <batchsize>128</batchsize>
+    <batchsize>4</batchsize>
 
     <!-- Scale factor is the number of wikipages *1000 -->
     <scalefactor>1</scalefactor>
 
+    <!-- Limit on the size of the string inserted into the OLD_TEXT column -->
+    <oldtextmaxlen>1048576</oldtextmaxlen>
```

Comment:
the challenge with this approach is the xml and ultimate WorkloadConfiguration class is used by every benchmark.  This property only makes sense for wikipedia.  I'd like to find a solution thats more generic.  One thought is to handle this in the ddl for cockroach by making the column length smaller and inspecting the column length in the java code for the loader.  Another approach might be to create a facility for benchmark specific customizations.  Lets chat.

Reply:
@timveil, after reading up a bit on this CockroachDB setting, `sql.conn.max_read_buffer_message_size`, it seems like the best approach to overcoming this 16 MB limitation per statement.  I've just dialed that up to 256 MB and am seeing fine results with a Wikipedia batch size of 32, so I think we can disregard this PR.

Thanks for having a look at it!